node/core/pvf/binary-builder/src/version.rs

```diff
@@ -0,0 +1,197 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+use std::cmp::Ordering;
+
+/// The version of rustc/cargo.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub struct Version {
```

Comment:
You don't need anything of this, as you don't need to check for any paritcular version of rust. We just added this to support compiling wasm on stable. 

TLDR: Please remove.

Reply:
Yeah, I left in a lot of code from wasm-builder with TODOs. My idea was, to do minimal changes for (hopefully) easier diffing with `wasm-builder`, and to maybe make the builder easier to generalize later. If it's not helpful for reviewers, I can remove.